pkg/models/appconfiguration/workload/container/container.go

```diff
@@ -44,16 +68,11 @@ type Probe struct {
 	FailureThreshold int32 `yaml:"failureThreshold,omitempty" json:"failureThreshold,omitempty"`
 }
 
-type ProbeType struct {
-	// Type of action to be taken.
-	Type string `yaml:"_type" json:"_type"`
-}
-
 // ProbeHandler defines a specific action that should be taken in a probe.
 // One and only one of the fields must be specified.
 type ProbeHandler struct {
 	// Type of action to be taken.
-	ProbeType `yaml:"_type" json:"_type"`
+	ActionType `yaml:"_type" json:"_type"`
```

Comment:
Can you explain the reason behind renaming `ProbeType` to `ActionType`? According to my understanding, `_type` is the schema name of kusion models, maybe you can just name this field as `Type`.

Reply:
To make YAML decoder happy, we have to add a type wrapper struct, and for Golang anonymous field requirement, we cannot name this wrapper struct as `Type`